packages/perseus/src/__tests__/server-item-renderer.test.tsx

```diff
@@ -271,6 +272,85 @@ describe("server item renderer", () => {
         expect(onRendered).toHaveBeenCalledWith(true);
     });
 
+    it("should call the onRendered callback when if no assets in content", () => {
+        const content: PerseusItem = {
+            question: {
+                content:
+                    "\n**What is the meaning of *intrigued*, as it is used in paragraph 2?**\n\n[[☃ radio 1]]\n\n\n\n\n\n",
+                images: {},
+                widgets: {
+                    "radio 1": {
+                        type: "radio",
+                        alignment: "default",
+                        static: false,
+                        graded: true,
+                        options: {
+                            choices: [
+                                {
+                                    content: "inventive",
+                                    correct: false,
+                                },
+                                {
+                                    content: "easily distracted",
+                                    correct: false,
+                                },
+                                {
+                                    content: "very interested",
+                                    correct: true,
+                                },
+                                {
+                                    content: "competitive",
+                                    correct: false,
+                                },
+                            ],
+                            randomize: false,
+                            multipleSelect: false,
+                            countChoices: false,
+                            displayCount: null,
+                            hasNoneOfTheAbove: false,
+                            deselectEnabled: false,
+                            numCorrect: 1,
+                        },
+                        version: {
+                            major: 2,
+                            minor: 0,
+                        },
+                    },
+                },
+            },
+            answerArea: {
+                zTable: false,
+                calculator: false,
+                chi2Table: false,
+                financialCalculatorMonthlyPayment: false,
+                financialCalculatorTotalAmount: false,
+                financialCalculatorTimeToPayOff: false,
+                periodicTable: false,
+                periodicTableWithKey: false,
+                tTable: false,
+            },
+            hints: [],
+        };
```

Comment:
Just because I'm saying this to everyone: I would love if we could make the simplest test objects possible when writing tests. This is long and it's unclear what's important for the same of the test.

```suggestion
const content: PerseusItem = {
    question: {
        content: "[[☃ radio 1]]",
        images: {},
        widgets: {
            "radio 1": {
                type: "radio",
                options: {
                    choices: [
                        {
                            content: "Correct",
                            correct: true,
                        },
                        {
                            content: "Incorrect",
                            correct: false,
                        },
                    ],
                },
            },
        },
    },
    answerArea: null,
    hints: [],
};
```

This is shorter and will have the same impact I think. Even this isn't particularly clear (we're not explicitly making an object without assets), but at least it's easier to see there's one widget, it has no assets, and there's nothing hiding in the content.